www/co-pixelStrings.php

```diff
@@ -1362,6 +1367,61 @@ function ValidateBBBStrings(data) {
     }
 }
 
+//map port# to user-friendly pin name + header name:
+const port2pin = {
+    0: "GPIO4 (P1-7)",
+    1: "GPIO5 (P1-29)",
+    2: "GPIO6 (P1-31)",
+    3: "GPIO7 (P1-26)",
+    4: "GPIO8 (P1-24)",
+    5: "GPIO9 (P1-21)",
+    6: "GPIO10 (P1-19)",
+    7: "GPIO11 (P1-23)",
+
+    8: "GPIO12 (P1-32)",
+    9: "GPIO13 (P1-33)",
+    10: "GPIO14 (P1-8)",
+    11: "GPIO15 (P1-10)",
+    12: "GPIO16 (P1-36)",
+    13: "GPIO17 (P1-11)",
+    14: "GPIO18 (P1-12)",
+    15: "GPIO19 (P1-35)",
+
+    16: "GPIO20 (P1-38)",
+    17: "GPIO21 (P1-40)",
+    18: "GPIO22 (P1-15)",
+    19: "GPIO23 (P1-16)",
+    20: "GPIO24 (P1-18)",
+    21: "GPIO25 (P1-22)",
+    22: "GPIO26 (P1-37)",
+    23: "GPIO27 (P1-13)",
+};
```

Comment:
This information needs to be pulled from the cape JSON.   Different capes/hats can use different pins for each port and thus this map would not be correct.

Reply:
The "GPIO" piece is not in the capes.  Is it okay if I add that into the pin names in the cape .sjon file?

Reply:
No, just the pin names in there....   For now, I would change this to a map like:

```
const PIN2GPIO = {
  "P1-7": "GPIO4",
  "P1-29: "GPIO5",
..
}
```
and use that to create the needed string.   Ideally, even this would be pulled via the GPIO rest API: 

http://fpp/api/gpio

Reply:
Okay, I'll change it to use /api/gpio.  I didn't see that one listed in REST api help, but it gives exactly the info I wanted.  thanks!